.pre-commit-config.yaml

```diff
@@ -834,12 +834,12 @@ repos:
         entry: ./scripts/ci/pre_commit/pre_commit_ui_lint.py
         pass_filenames: false
         additional_dependencies: ['yarn@1.22.19']
-      - id: lint-javascript
+      - id: ts-compile-and-lint-javascript
         name: ESLint against current UI JavaScript files
```

Comment:
Let's change the description too.

Reply:
Yes I didn't change it cause we are limited in length :(, trying to find something better

Reply:
`Generat and lint UI Javascript files`  ?

Reply:
I went for `TS types generation and ESLint against current UI files`